external/anomaly/ote_tests_pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+python_files = test_ote_anomaly_segmentation_training.py
```

Comment:
```suggestion
python_files = test_ote_*.py
```

Reply:
(You could commit this at the end of developing, -- I understand that this trick increases speed of test jobs)